cmake/clang-format.cmake

```diff
@@ -34,13 +34,17 @@ if(NOT CLANG_FORMAT_BIN)
     endif()
   endif()
 endif()
-
+set(CLANG_FORMAT_BIN clang-format)
```

Comment:
it's better to make it an option with default value

Reply:
I was trying to fix a bug in this code but I better provide the fix in a separate PR.